pkg/kubectl/cmd/apply_set_last_applied.go

```diff
@@ -200,46 +197,13 @@ func (o *SetLastAppliedOptions) RunSetLastApplied(f cmdutil.Factory, cmd *cobra.
 			cmdutil.PrintSuccess(o.Mapper, o.ShortOutput, o.Out, info.Mapping.Resource, info.Name, o.DryRun, "configured")
 
 		} else {
-			err := o.formatPrinter(o.Output, patch.Patch, o.Out)
+			out, err := diff.FormatConvert(patch.Patch, o.Output)
 			if err != nil {
 				return err
 			}
+			fmt.Fprintf(o.Out, string(out))
```

Comment:
no, this value should pass to PrintSuccess, if we execute `./kubectl apply set-last-applied -f ~/nginx.yaml  -o yaml` it will print out